tests/test_hbavss.py

```diff
@@ -603,6 +647,12 @@ def _get_dealer_msg(self, values, n):
 
 @mark.asyncio
 async def test_hbavss_light_client_mode(test_router):
+    def callback(future):
```

Comment:
We can now import this as:
`from honeybadgermpc.utils.misc import print_exception_callback`

Reply:
oh cool. that's a good change